index.html

```diff
@@ -302,7 +302,7 @@ <h4><a href="./news/2019-07-02-Guelph-accessibility-conference.html">Floe at the
 
                 </div>
             </div>
-
+            <br><br>
```

Comment:
Spaces should be done with CSS styles not using `<br>` tags. That is, the markup is to structure the content, CSS is to style it.

Reply:
ok i will make the changes!